Validation/SiTrackerPhase2V/plugins/Phase2ITValidateCluster.cc

```diff
@@ -312,17 +312,17 @@ void Phase2ITValidateCluster::fillDescriptions(edm::ConfigurationDescriptions& d
   {
     edm::ParameterSetDescription psd0;
     psd0.add<std::string>("name", "Delta_X_Pixel");
-    psd0.add<std::string>("title", "#Delta X;Cluster resolution X dimension");
+    psd0.add<std::string>("title", "#Delta X;Cluster resolution X coordinate");
```

Comment:
can units be put here?

Reply:
@mmusich the range can be certainly reduced, the only doubt I have is that all the layers (rings) share these ranges and if we cut further we will cutting the tail around the peak in zero for some of the layers (rings). So the question is: How important is to keep the tails?
PS. We are adding the units now.

Reply:
> will cutting the tail around the peak in zero for some of the layers (rings). So the question is: How important is to keep the tails?

do we have concrete example to judge that? I think we're vastly more interested in the core of the residual. If the tail is substantial can it be put in the overflow bin?

Reply:
For example this: 
![image](https://user-images.githubusercontent.com/50108150/110624653-cbdce780-819e-11eb-8f0b-8d0d7d506c53.png)
Good, we will the reduce the range. Is [-0.5,0.5] a sensible range for you?

Reply:
>  Good, we will the reduce the range. Is [-0.5,0.5] a sensible range for you?

seems a sensible range to me. OTOH the plot you posted above worries me. Are the units cm? How comes to have the closest cluster 5 cm away from the simHit, that seems excessive. Are those only the primaries or do delta-rays enter here?

Reply:
@emiglior FYI

Reply:
I think the distribution clearly looks bimodal. I suppose there is a core of residuals related to the right simHit-cluster association superimposed to a random combinatorics background of simHits associated to a semi-randomly chosen cluster, probably because the right cluster was never produced. In this sense the tails are "tracking" the clusterization inefficiency and are not representative of the detector resolution. IIRC we already discussed about imposing a cut on the association window of few channels maximum. The tails are kind of interesting but don't belong here IMHO (could be diverted to another plot).

Reply:
It is meant to be big enough number that so it does not leave anything out. Do you think a smaller number can be defined that could make filter out some of the simhits that are far away?

Reply:
Definitely. If we leave matches to be as large as 10m, clearly there will be some some residuals which don't make sense.
So, either that or matching both from x and y directions at the same time, I think that @sroychow was suggesting this latter approach.

Reply:
In principle you have also the y direction for the macro-pixels in the P-sensor of PS modules and some stricter criteria for y for the S-sensor of SS and PS modules can be used.